### ISSUE TITLE
How to add skin

### ISSUE THREAD

**Issue by paladox** [kaltura/mwEmbed#1922]
Hi how can I add the skin for the player since the old default was removed for version 2. But the skin that is included in v2 is basic doesent support control panel or doesent do what the prevous skin did. Where do I get the skin and how can I update it for TimedMediaHanlder and MwEmbedSupport for mediawiki. Since some of the required files are in different locations in this repo now. not sure which ones to include.

**Comment by kaltura-hooks** [comment]
Hi @paladox,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
 - Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
    please provide as much details as possible.
 - Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
	If self hosted, are you using the RPM or deb install?
 - Packages installed.
	When using RPM, paste the output for:
```
	# rpm -qa "kaltura*"
```	
	For deb based systems:
```	
	# dpkg -l "kaltura-*"
```
 - If running a self hosted ENV - provide the MySQL server version used
 - If running a self hosted ENV - is this a single all in 1 server or a cluster?
 - If running a self hosted ENV, while making the problematic request, run:
```
	# tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color "ERR:\|PHP\|trace\|CRIT\|\[error\]"
```
   and paste the output.
 - When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by amirch1** [comment]
Hi,
The Kaltura V2 player skin files reside in the kdark folder under skins: https://github.com/kaltura/mwEmbed/tree/master/skins/kdark
Basically in consists of the following:
1. Player icons specified in the file css/icons.css. The icons reside in the font file: fonts/icomoon.svg that can be edited in the iconmoon application
2. css/layout.css contains all the player look and feel classes

In order to customise the player style without touching the code you can either:
1. Use the "theme" plugin to change color and size or all player components. You can do this visually in Universal Studio
2. Write your own custom CSS that overrides classes from our layout.css file and load it in the player UIConf. 

Please let me know if you need any additional assistance.
Amir

**Comment by paladox** [comment]
How could I add it. Since Timedmediahandler handles the embedplayerand timedtext. And mwemedsupport handles what the name says. But I am having a hard time loading the css  for the player since it works in v1 but v2 no.and so when I load the player it loads but doesent show any control bar or even a play button just plain video player with just the video image then you click it to play and click it again to pause.

**Comment by amirch1** [comment]
Can you send me a test page showing what you described?

**Comment by paladox** [comment]
http://en.random-wikisaur.tk/wiki/File:Zooming_into_the_southern_spiral_NGC_300_(ESO_1037b).webm

**Comment by paladox** [comment]
Wrong link

**Comment by paladox** [comment]
http://simple-random-wikisaur.tk/index.php?title=File:Zooming_into_the_southern_spiral_NGC_300_(ESO_1037b).webm&mobileaction=toggle_view_desktop

**Comment by amirch1** [comment]
@mdale  - how does "in-progress-state" sounds?

**Comment by amirch1** [comment]
I don't see the Kaltura player loading here - only a video tag with its native controls. Are you using the Kaltura embed code at all?

**Comment by paladox** [comment]
Yes but i am doing it in the format of the extension meaning that i am not using the full download of kaultra just the embedplayer and timedtext and mwemebedsupport folders. I am not sure how to get skin to work since it is not the default any more.

**Comment by amirch1** [comment]
@itaykinnrot, @mdale   - I'm not familiar with this workflow. Can you please advise?

**Comment by paladox** [comment]
I belive that kaultra and Wikimedia were developing a video player for Wikimedia use like for Wikipedia which does but it needs to be upgraded to stay current in the technology these days.

**Comment by mdale** [comment]
Paladox, this will be a bit more complicated. Timed media handler would need to bring in a the "KalturaSupport" module into TMH, have a way for TMH to supply entries to player via embed services or equivalent.  https://github.com/kaltura/embed-services or via JSON mediaProxy override.  This is because the kaltura player has more dependencies on normalized metadata. You can see all the stuff that goes into instantiation of a player this page: 
http://player.kaltura.com/modules/KalturaSupport/tests/RuntimeOverrides.html

**Comment by TF2CutContentWiki** [comment]
Hate to revive such an old issue, but we use the Kaltura player over on TCRF.net and I've made [a dark mode skin for TCRF](https://github.com/TF2CutContent/TCRFDark), but have not yet messed around with the Kaltura player's styling as I have a question:

I am **not** the host/owner of TCRF and as such, do not have access to the internal files of the wik. I assume it should be possible to affect the styling of the media player through one's "[User:<name>/common.css](https://tcrf.net/User:TF2CutContent/common.css)". Am I correct in my assumption?
